lib/src/modules/instabug.dart

```diff
@@ -393,10 +393,32 @@ class Instabug {
 
   /// Sets the repro steps mode
   /// [mode] repro steps mode
+  @Deprecated('Use [setReproStepsConfig] instead.')
   static Future<void> setReproStepsMode(ReproStepsMode reproStepsMode) async {
     return _host.setReproStepsMode(reproStepsMode.toString());
   }
 
+  /// Sets the repro steps mode for bugs and crashes.
+  /// [bug] repro steps mode for bug reports.
+  /// [crash] repro steps mode for crash reports.
+  /// [all] repro steps mode for both bug and crash reports, when present it
+  /// overrides [bug] and [crash].
```

Comment:
@a7medev What do you think if we add an example here, just for reference.

Reply:
Looks good but let's remove the `all` parameter from the example since it overrides the other two.

Reply:
Awesome, here's the final doc for the method.

```dart

```dart
    /// Sets the repro steps mode for bugs and crashes.
    ///
    /// [bug] repro steps mode for bug reports.
    /// [crash] repro steps mode for crash reports.
    /// [all] repro steps mode for both bug and crash reports, when present it
    /// overrides [bug] and [crash].
    ///
    /// Example:
    /// ```dart
    /// Instabug.setReproStepsConfig(
    ///  bug: ReproStepsMode.enabledWithNoScreenshots,
    ///  crash: ReproStepsMode.enabled,
    ///  );
    static Future<void> setReproStepsConfig({
  ```
  
  I've changed the value of the `crash` param to `ReproStepsMode.enabled` to make more sense.

Reply:
Just one more suggestion, let's make sure the example is properly formatted and let's add the triple ` at the end of the example code.

Reply:
Last final getsuga ten.. i mean change 🤣 

```dart
    /// Sets the repro steps mode for bugs and crashes.
    ///
    /// [bug] repro steps mode for bug reports.
    /// [crash] repro steps mode for crash reports.
    /// [all] repro steps mode for both bug and crash reports, when present it
    /// overrides [bug] and [crash].
    ///
    /// Example:
    /// ```dart
    /// Instabug.setReproStepsConfig(
    ///  bug: ReproStepsMode.enabledWithNoScreenshots,
    ///  crash: ReproStepsMode.enabled,
    ///  );
    ///  ```
    static Future<void> setReproStepsConfig({
  ```